src/libs/actions/Report.ts

```diff
@@ -851,7 +851,8 @@ function navigateToAndOpenChildReport(childReportID = '0', parentReportAction: P
         const newChat = ReportUtils.buildOptimisticChatReport(
             participantAccountIDs,
             parentReportAction?.message?.[0]?.text,
-            parentReport?.chatType,
+            // Threads from DMs and selfDMs don't have a chatType. All other threads inherit the chatType from their parent
+            parentReport && ReportUtils.isSelfDM(parentReport) ? undefined : parentReport?.chatType,
```

Comment:
Can we use a variable for this instead, as I recommended here: https://expensify.slack.com/archives/C01GTK53T8Q/p1713809961011049?thread_ts=1713763386.882099&cid=C01GTK53T8Q

You can define it below parentReport on line 850.

Reply:
@puneetlath I just updated